Ansible/ansible_collections/jfrog/platform/roles/missionControl/files/searchguard/localhost.pem

```diff
@@ -0,0 +1,51 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We should not have .pem files in a git repo.

Reply:
same as above